config/production.yml

```diff
@@ -2,8 +2,9 @@ accounts:
   passwordResetEmailEnabled: true
 
 defaultOTP: '1111'
```

Comment:
We can use `otp` instead for OTP level settings. May be we can do `otp.defaultPhoneNumber`, `otp.defaultOTP` ?

Reply:
I have been rethinking the approach. Let's just enable default OTP and PhoneNumber for all instances for now (not just with prod). We can this to default config instead.

Reply:
Sure, will update it with this approach.